run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-docker run -d --name orb-update -p 8000:80 -v $(pwd):/var/www/html orb-status
+docker run -d --name orbs-status -p 8000:80 -v $(pwd):/var/www/html orb-status
```

Comment:
Super minor, but maybe we should build the image as named `orbs-status` as well to align with the repo name?